yarn.lock

```diff
@@ -909,7 +909,7 @@
 
 "@digicat/components@file:/Users/pmichelevicius/workspace/ui-component-library/packages/components":
```

Comment:
can we remove this yarn.lock?

Reply:
Can do, It's also removed in the following pr